.drone.yml

```diff
@@ -84,6 +84,25 @@ pipeline:
       local: false
       event: [ push ]
 
+  update-search-index:
+    image: algolia/docsearch-scraper
+    entrypoint: ""
+    pull: true
+    secrets: [ 
```

Comment:
Please use one or another approach for the formatting of array values, only one of these:

```
secrets: [ a, b, c ]
```

```
secrets:
- a
- b
- c
```

Reply:
Just trying to make it look neat and tidy.